Thu Jan 17 13:46:37 CST 2019
Command: /home/bin/driver_new /home/bin/config.sj 
========================================================================
========================================================================
Boot Time: Thu Jan 17 13:46:37 2019
FluffOS Version: fluffos v2019.2019011401(CMake/Broken)@ (Linux/x86-64)
========================================================================
Core Dump: Yes, Max FD: 16384
*Warning: obsolete line in config file, please delete:
	address server ip 
*Warning: obsolete line in config file, please delete:
	address server port 
*Warning: obsolete line in config file, please delete:
	reserved size 
Processing config file: /home/bin/config.sj
==== Runtime Config Table ====
time to clean up: 600 (new: 900)
time to reset: 900
time to swap: 300 (new: 0)
evaluator stack size: 65536 (new: 1000)
inherit chain size: 30 (new: 60)
maximum evaluation cost: 30000000 (new: 50000000)
maximum local variables: 50 (new: 60)
maximum call depth: 150 (new: 100)
maximum array size: 15000 (new: 30000)
maximum buffer size: 400000
maximum mapping size: 150000 (new: 30000)
maximum string length: 200000
maximum bits in a bitfield: 12000 (new: 1200)
maximum byte transfer: 200000 (new: 10000)
maximum read file size: 200000
hash table size: 7001
object table size: 1501
living hash table size: 256 (new: 100)
gametick msec: 1000
heartbeat interval msec: 1000
sane explode string: 1
reversible explode string: 0
sane sorting: 1
warn tab: 0
wombles: 0
call other type check: 0
call other warn: 0
mudlib error handler: 1
no resets: 0
lazy resets: 0
randomized resets: 1
no ansi: 1
strip before process input: 1
this_player in call_out: 1
trace: 1
trace code: 0
interactive catch tell: 0
receive snoop: 1
snoop shadowed: 0
reverse defer: 0
has console: 1
noninteractive stderr write: 0
trap crashes: 1
old type behavior: 0
old range behavior: 0
warn old range behavior: 1
suppress argument warnings: 1
enable_commands call init: 1
sprintf add_justified ignore ANSI colors: 1
call_out(0) nest level: 1000
==============================
adm/log/debug.log: No such file or directory
Event backend in use: epoll
==== LPC Predefines ====
#define FLUFFOS 
#define HAS_DEBUG_LEVEL 
#define HAS_ED 
#define HAS_PRINTF 
#define MAX_FLOAT 179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000
#define MAX_INT 9223372036854775807
#define MIN_FLOAT 0.000000
#define MIN_INT -9223372036854775808
#define MUDOS 
#define MUD_NAME "书剑・怀旧站"
#define SIZEOFINT 8
#define __ARCH__ "Linux/x86-64"
#define __ARGUMENTS_IN_TRACEBACK__ 
#define __ARRAY_STATS__ 
#define __CACHE_STATS__ 
#define __CALLOUT_HANDLES__ 
#define __CFG_COMPILER_STACK_SIZE__ 600
#define __CFG_EVALUATOR_STACK_SIZE__ 65536
#define __CFG_LIVING_HASH_SIZE__ 256
#define __CFG_MAX_CALL_DEPTH__ 150
#define __CFG_MAX_GLOBAL_VARIABLES__ 65536
#define __CFG_MAX_LOCAL_VARIABLES__ 50
#define __CLASS_STATS__ 
#define __COMMAND_BUF_SIZE__ 2000
#define __COMPILER__ "/usr/bin/c++"
#define __CUSTOM_CRYPT__ 
#define __CXXFLAGS__ "Broken"
#define __DEBUG_MACRO__ 
#define __DEFAULT_PRAGMAS__ PRAGMA_WARNINGS + PRAGMA_SAVE_TYPES + PRAGMA_ERROR_CONTEXT + PRAGMA_OPTIMIZE
#define __ED_INDENT_SPACES__ 4
#define __ED_TAB_WIDTH__ 8
#define __GET_CHAR_IS_BUFFERED__ 
#define __HAS_CONSOLE__ 
#define __LARGEST_PRINTABLE_STRING__ 8192
#define __LARGE_STRING_SIZE__ 1000
#define __LOCALS_IN_TRACEBACK__ 
#define __MAX_SAVE_SVALUE_DEPTH__ 100
#define __MUDLIB_ERROR_HANDLER__ 
#define __OLD_ED__ 
#define __PACKAGES_PACKAGES_H__ 
#define __PACKAGE_COMPRESS__ 
#define __PACKAGE_CONTRIB__ 
#define __PACKAGE_CORE__ 
#define __PACKAGE_CRYPTO__ 
#define __PACKAGE_DB__ 
#define __PACKAGE_DEVELOP__ 
#define __PACKAGE_MATH__ 
#define __PACKAGE_MATRIX__ 
#define __PACKAGE_MUDLIB_STATS__ 
#define __PACKAGE_OPS__ 
#define __PACKAGE_PARSER__ 
#define __PACKAGE_PCRE__ 
#define __PACKAGE_SHA1__ 
#define __PACKAGE_SOCKETS__ 
#define __PACKAGE_TRIM__ 
#define __PACKAGE_UIDS__ 
#define __PORT__ 7777
#define __RANDOMIZED_RESETS__ 
#define __RECEIVE_SNOOP__ 
#define __REF_RESERVED_WORD__ 
#define __RESTRICTED_ED__ 
#define __SANE_EXPLODE_STRING__ 
#define __SANE_SORTING__ 
#define __SAVE_EXTENSION__ ".o"
#define __SENSIBLE_MODIFIERS__ 
#define __SMALL_STRING_SIZE__ 100
#define __STRING_STATS__ 
#define __STRUCT_CLASS__ 
#define __STRUCT_STRUCT__ 
#define __SUPPRESS_ARGUMENT_WARNINGS__ 
#define __THIS_PLAYER_IN_CALL_OUT__ 
#define __TRACE__ 
#define __TRAP_CRASHES__ 
#define __USE_32BIT_ADDRESSES__ 
#define __VERSION__ "fluffos v2019.2019011401 (CMake/Broken)"
#define __WARN_OLD_RANGE_BEHAVIOR__ 
========================

Loading preload files ...
/adm/daemons/securityd...
/adm/daemons/virtuald...
/adm/daemons/rankd...
/adm/daemons/commandd...
/adm/daemons/chinesed...
/adm/daemons/emoted...
/adm/daemons/channeld...
/adm/daemons/aliasd...
/adm/daemons/fingerd...
/adm/daemons/natured...
/adm/daemons/network/dns_master...
/adm/daemons/logind...
/adm/daemons/questd.c...
/adm/daemons/cron.c...
Accepting connections on 0.0.0.0:7777.
Initializations complete.

错误讯息被拦截: 
执行时段错误：*Bad argument 1 to explode()
Expected: string Got: 0.

程式：/adm/daemons/questd.c 第 749 行
物件: /adm/daemons/questd ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/adm/daemons/questd.c 的 create() 第 39 行，物件： /adm/daemons/questd ("0(0)")
呼叫来自：/adm/daemons/questd.c 的 read_table() 第 749 行，物件： /adm/daemons/questd ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/user_weapond'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/user_weapond'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/user_weapond'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/update.c 第 64 行
物件: /cmds/adm/update ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/update.c 的 main() 第 64 行，物件： /cmds/adm/update ("0(0)")
呼叫来自：/cmds/adm/update.c 的 CATCH() 第 64 行，物件： /cmds/adm/update ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/update.c 第 64 行
物件: /cmds/adm/update ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/update.c 的 main() 第 64 行，物件： /cmds/adm/update ("0(0)")
呼叫来自：/cmds/adm/update.c 的 CATCH() 第 64 行，物件： /cmds/adm/update ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/update.c 第 64 行
物件: /cmds/adm/update ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/update.c 的 main() 第 64 行，物件： /cmds/adm/update ("0(0)")
呼叫来自：/cmds/adm/update.c 的 CATCH() 第 64 行，物件： /cmds/adm/update ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/taskd'

程式：/cmds/adm/update.c 第 64 行
物件: /cmds/adm/update ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/update.c 的 main() 第 64 行，物件： /cmds/adm/update ("0(0)")
呼叫来自：/cmds/adm/update.c 的 CATCH() 第 64 行，物件： /cmds/adm/update ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/user_weapond'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/feature/vi'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误
执行时段错误：*Bad argument 1 to new()
Expected: string Got: 0.

程式：/adm/daemons/logind.c 第 583 行
物件: /adm/daemons/logind ("0(0)")
对象：/clone/user/login#307 ("0(Cicoo)")
呼叫来自：/adm/daemons/logind.c 的 get_passwd() 第 307 行，物件： /adm/daemons/logind ("0(0)")
呼叫来自：/adm/daemons/logind.c 的 make_body() 第 583 行，物件： /adm/daemons/logind ("0(0)")
错误
执行时段错误：*Bad argument 1 to new()
Expected: string Got: 0.

程式：/adm/daemons/logind.c 第 583 行
物件: /adm/daemons/logind ("0(0)")
对象：/clone/user/login#416 ("0(Jick)")
呼叫来自：/adm/daemons/logind.c 的 get_passwd() 第 307 行，物件： /adm/daemons/logind ("0(0)")
呼叫来自：/adm/daemons/logind.c 的 make_body() 第 583 行，物件： /adm/daemons/logind ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#64 ("咩绝(Mie)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#64 ("咩绝(Mie)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#21 ("马芫(Maxin)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#21 ("马芫(Maxin)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#80 ("天下无狗(Wugou)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#80 ("天下无狗(Wugou)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#43 ("小糖(Riri)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#43 ("小糖(Riri)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#43 ("小糖(Riri)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#43 ("小糖(Riri)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#43 ("小糖(Riri)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#43 ("小糖(Riri)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#149 ("紫玉(Ziyu)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#149 ("紫玉(Ziyu)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#8 ("打工仔(Bbb)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#8 ("打工仔(Bbb)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#43 ("小糖(Riri)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#43 ("小糖(Riri)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/feature/command.c 的 command_hook() 第 62 行，物件： /clone/user/user#319 ("青铜(Qingtong)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#8 ("打工仔(Bbb)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#8 ("打工仔(Bbb)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#43 ("小糖(Riri)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#43 ("小糖(Riri)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#43 ("小糖(Riri)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#43 ("小糖(Riri)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#387 ("夜叉(Yecha)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/go.c 第 84 行
物件: /cmds/std/go ("0(0)")
对象：/clone/user/user#149 ("紫玉(Ziyu)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#149 ("紫玉(Ziyu)")
呼叫来自：/cmds/std/go.c 的 main() 第 84 行，物件： /cmds/std/go ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/adm/daemons/logind.c 的 get_gender() 第 574 行，物件： /adm/daemons/logind ("0(0)")
呼叫来自：/adm/daemons/logind.c 的 enter_world() 第 658 行，物件： /adm/daemons/logind ("0(0)")
呼叫来自：/feature/move.c 的 move() 第 176 行，物件： /clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#447 ("探路者(Tanlu)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误讯息被拦截: 
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/adm/mup.c 第 64 行
物件: /cmds/adm/mup ("0(0)")
对象：/clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#1 ("工作人员(Anger)")
呼叫来自：/cmds/adm/mup.c 的 main() 第 64 行，物件： /cmds/adm/mup ("0(0)")
呼叫来自：/cmds/adm/mup.c 的 CATCH() 第 64 行，物件： /cmds/adm/mup ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#8 ("打工仔(Bbb)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#8 ("打工仔(Bbb)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
错误
执行时段错误：*Error in loading object '/adm/daemons/natured'

程式：/cmds/std/look.c 第 31 行
物件: /cmds/std/look ("0(0)")
对象：/clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/feature/command.c 的 command_hook() 第 66 行，物件： /clone/user/user#433 ("乳酪(Rulao)")
呼叫来自：/cmds/std/look.c 的 main() 第 31 行，物件： /cmds/std/look ("0(0)")
closed external ports
clear_call_outs: 17 leftover callouts cleared.
clear_tick_events: 31 leftover events cleared.
Initializing internal stuff ....
